deploy/openshift/sws.yaml

```diff
@@ -1,10 +1,9 @@
-id: sws
-kind: Template
 apiVersion: v1
-name: SWS Template
+kind: Template
```

Comment:
We probably we will to reconsider using a template in the future (its an OpenShift only component and wont work with Kubernetes). But for now it should be good.

Reply:
Right, I'm not concerned with k8s right now as the team dictate for now has been "get it on openshift". using a template is nice because we can pass in parameters via env vars to configure the components (port, docker version, verbose mode, etc).